### PR TITLE
fix: add console logo 😅

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- On popular demand..

Screenshot 2023-12-06 at 14 47 42

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]
Awesome! Though, you can there's colours in the console too? 😈

File: packages/shared/src/hooks/useConsoleLogo.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+               ....             ..
+
+
+
+
+
+
+                                                  `);
+  }, []);
+}
```
Comment (capJavert) [inline packages/shared/src/hooks/useConsoleLogo.ts:32]: Let's not have it in development please 🙏

**Review by capJavert — COMMENTED** [review]
Ahaha awesome 👏

**Review by capJavert — APPROVED** [review]
I love the logo!

**Pull request merged**